.all-contributorsrc

```diff
@@ -1,6 +1,7 @@
 {
   "files": [
-    "README.md"
+    "README.md",
```

Comment:
Can I recommend that we add a link to the configuration docs as a comment to the top of this file so that folks working on this stuff know where to go for info? As I had to look up the `files` key to understand that it's purpose and your intention matched 😄 

Relevant link is: https://allcontributors.org/docs/en/bot/configuration

Reply:
Good point, I should do the same in #3102.

Turns out JSON doesn't have comments in the same way as programming languages. I added a key called `_comment` with the information and the CLI doesn't complain and still parses the file.

Fixed in f8b673b.

Reply:
Did `//` not work? What failed exactly?

Reply:
With the CLI `//` does work, so I would guess that works with the bot too. I think it is a bit risky though as it is not valid JSON so a parser could quite reasonably return an error.

That said, there is nothing special about an underscore to stop a program reading and using `_comment` :shrug:.